### PR TITLE
Add json

### PR DESCRIPTION
**Pull request by glstewart17**
This branch completes week 3 part 3.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -14,19 +14,57 @@
...
+  private List<QuotePerson> quotes;
+
+  /**
+   * Add all the quotes to quotes using type QuotePerosn on start.
+   */
+  @Override
+  public void init() {
+    quotes = new ArrayList<>();
+    quotes.add(new QuotePerson("Bears. Beets. Battlestar Galactica.", "Jim Halpert"));
+    quotes.add(new QuotePerson("I'm not supersitious, but I am a little stitious.", "Michael Scott"));
```
Comment (aditij113) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:40]: LOL just saw this episode a few days ago.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -14,19 +14,57 @@
...
 public class DataServlet extends HttpServlet {
 
+  private List<QuotePerson> quotes;
+
+  /**
+   * Add all the quotes to quotes using type QuotePerosn on start.
+   */
+  @Override
+  public void init() {
+    quotes = new ArrayList<>();
```
Comment (aditij113) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:38]: This needs a Dwight quote.

quotes.add(new QuotePerson("“You couldn’t handle my undivided attention.”, "Dwight Schrute"));

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (glstewart17) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:40]: It's a great quote!

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (glstewart17) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:38]: Thanks, I just added it.

**Pull request merged**